modules/dkan/dkan_dataset/modules/dkan_dataset_collections/dkan_dataset_collections.features.filter.inc

```diff
@@ -0,0 +1,46 @@
+<?php
```

Comment:
I don't think we need this anymore, [DKAN should allow H2](https://github.com/GetDKAN/dkan/blob/7.x-1.x/modules/dkan/dkan_sitewide/dkan_sitewide.features.filter.inc#L31) which was I think the reason this was added?

Reply:
:+1: